src/SourceBuild/patches/wpf/0001-more-system-memory-fixes.patch

```diff
@@ -0,0 +1,37 @@
+From f704647788c026d99652b5b3716fab92486841d9 Mon Sep 17 00:00:00 2001
+From: Viktor Hofer <viktor.hofer@microsoft.com>
+Date: Tue, 21 May 2024 13:50:09 +0200
+Subject: [PATCH] Add missing System.Memory references (#9140)
+
+* Add missing System.Memory references
+
+* Update WindowsFormsIntegration.csproj
```

Comment:
Friendly reminder to always include a `Backport: <nnn>` line to reference the issue/PR for the patch.  This makes tracking  easier especially for those not involved in the creation.  https://github.com/dotnet/source-build/blob/main/Documentation/patching-guidelines.md#patch-guidelines